docs/get-started.md

```diff
@@ -26,13 +26,45 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
 
 ### Set up server
 
-1. Start the server and the local storage service. In the root directory of Ark, run:
+These instructions assume that you are running Minio inside your cluster. They should be used for a test environment or to explore Ark only. Service of type `NodePort` is not recommended for production.
```

Comment:
`Service of type NodePort is not recommended for production.` might be worded a bit too strongly/broadly? I think it might be better to say that configuring Minio for a production-quality setup is out of scope here (e.g. you'll want to configure Minio for backups / high availability).